build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -244,8 +244,7 @@ flowExporter:
 nodePortLocal:
 {{- with .Values.nodePortLocal }}
 # Enable NodePortLocal, a feature used to make Pods reachable using port forwarding on the host. To
-# enable this feature, you need to set "enable" to true, and ensure that the NodePortLocal feature
-# gate is also enabled (which is the default).
+# enable this feature, you need to set "enable" to true.
```

Comment:
You didn't remove the feature gate from line 30-31.

Reply:
Please update the docs/feature-gates.md about NodePortLocal, and check if there is any description need to be updated on docs/node-port-local.md

Reply:
`feature-gates.md` and `node-port-local.md` updated.